tests/phpunit/integration/api/uikit/includes/class-uikit-test-case.php

```diff
@@ -117,13 +125,47 @@ protected function reset_globals() {
 			'themes'     => array(),
 		);
 
-		$_beans_uikit_registered_items = array(
-			'themes' => array(
+		$_beans_uikit_registered_items = [
```

Comment:
🍰 🥇 💯 for short array syntax

Reply:
Working on converting the rest over.  That would be a great first issue for someone to tackle.

Reply:
Yes. Let's open an issue.